polyglot/piranha/README.md

```diff
@@ -133,6 +133,7 @@ piranha_summary = run_piranha_cli(path_to_codebase,
   * `rules.toml`: *piranha rules* expresses the specific AST patterns to match and __replacement patterns__ for these matches (in-place). These rules can also specify the pre-built language specific cleanups to trigger.
   * `edges.toml` (_optional_): expresses the flow between the rules 
 - `dry_run` : Disables in-place rewriting of code 
+- `ignore_parse_error` : Does not throw an error on syntactically incorrect source code produced by a rewrite.
```

Comment:
Can we separate this change out to a different PR ?  This change is a bit tricky. What are we enabling? 
There are two things that can happen : 
1. We have a partially syntactically incorrect code as input (This can happen specially when our grammar does not support some language structure).
2. Our input code is syntactically correct but Piranha introduces a syntactic error 

For the former case we need to check if a new syntactic error is reported near the location of the most recent edit.

Reply:
I'm removing this change from the current PR; we can discuss it in another PR/issue.